lib/sdam/server_selection.js

```diff
@@ -5,7 +5,7 @@ const { MongoError } = require('../error');
 
 // max staleness constants
 const IDLE_WRITE_PERIOD = 10000;
-const SMALLEST_MAX_STALENESS_SECONDS = 20;
+const SMALLEST_MAX_STALENESS_SECONDS = 90;
```

Comment:
had to do this because of:
https://github.com/mongodb/specifications/blob/e56f5eceed7729f8b9b43a4a1f76c7e5840db49f/source/max-staleness/tests/ReplicaSetWithPrimary/MaxStalenessTooSmall.yml#L25